public/js/gamecore.js

```diff
@@ -62,10 +63,23 @@ $(document).ready(function()  {
                     for (var shot in attacksMade[attack]) {
                         // getting each towerID, creepID pair
                         // this is why we must send stuff as array of objects...
-                        drawShot(gameCtx, attack, attacksMade[attack][shot]);
+                        // drawShot(gameCtx, attack, attacksMade[attack][shot]);
+
+                        projectiles.push(
+                            new Projectile(gameCtx,
+                                attack,
+                                attacksMade[attack][shot],
+                                15));
```

Comment:
We probably don't want to be pushing new projectiles every render loop.

For back end projection control, we should set projectiles to what the server sends ([like here](https://github.com/zlailari/Towers/blob/ab45cd37db5b6931c273f0b81fad8d1533318154/public/js/Connection.js#L27) and just draw them. No need to worry about the splicing stuff below either because the server will remove it from the array is sends and front end will automatically have it updated